config/default/kustomization.yaml

```diff
@@ -0,0 +1,74 @@
+# Adds namespace to all resources.
+namespace: default
+
+# Value of this field is prepended to the
+# names of all resources, e.g. a deployment named
+# "wordpress" becomes "alices-wordpress".
+# Note that it should also match with the prefix (text before '-') of the namespace
+# field above.
+namePrefix: ibm-block-csi-operator-
+
+# Labels to add to all resources and selectors.
+#commonLabels:
+#  someName: someValue
+
+bases:
+- ../crd
+- ../rbac
+- ../manager
+# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in
+# crd/kustomization.yaml
+#- ../webhook
+# [CERTMANAGER] To enable cert-manager, uncomment all sections with 'CERTMANAGER'. 'WEBHOOK' components are required.
+#- ../certmanager
+# [PROMETHEUS] To enable prometheus monitor, uncomment all sections with 'PROMETHEUS'.
+#- ../prometheus
```

Comment:
keep or remove

Reply:
I wasn't referring to these specific lines, there are plenty of [them](https://github.com/IBM/ibm-block-csi-operator/blob/f5e9b2a9981e8df4fae7d46509e9f8e515b8dde4/config/crd/kustomization.yaml#L9), I meant in general...

Reply:
what about [these](https://github.com/IBM/ibm-block-csi-operator/blob/77e49620741523f247b3746146d76f5ac30c5230/config/default/kustomization.yaml#L7-L9)?

Reply:
removed them
done

Reply:
perhaps I wasn't clear - I'm not referring to [specific lines](https://github.com/IBM/ibm-block-csi-operator/blob/9d71a1be202ca892f4a691d3cd76777a7205e996/config/default/kustomization.yaml#L7-L9), I'm referring to this type of gap in the PR in general